eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -96,18 +95,17 @@ public void setFlagsFromResponse(Reader response) {
             flags = config.getFlags();
             Log.d(TAG, "Loaded " + flags.size() + " flags from configuration response");
         }
-
-        writeConfigToFile(config);
     }
 
-    private void writeConfigToFile(RandomizationConfigResponse config) {
+    public void asyncWriteToCache(String jsonString) {
```

Comment:
Promoted this to an upstream public function, as it seemed a weird side effect of a function that sets flags.

Reply:
yeah that's a good point; I'm keeping that pattern just separating the responsibilities of the underlying functions